apostrophecms/login-recaptcha
# PRO-2386 Beta

## Summary

Ports the login-recaptcha module from A2 to A3. Uses the new login requirements feature for A3

## What are the specific steps to test this change?

1. See the README for configuration if starting fresh. For internal review, use the `login-recaptcha` branch of the testbed project. Start the project
2. Make sure you're logged out and go to `http://localhost:3000/login`. Confirm that the reCAPTCHA logo is showing at bottom right (the main indicator this invisible thing is active).
3. Attempt to log in. Assuming you're human, this should work.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
